README.md

```diff
@@ -1,54 +1,77 @@
 # jupyterhub_moss: JupyterHub MOdular Slurm Spawner
 
-This package extends [`batchspawner.SlurmSpawner`](https://github.com/jupyterhub/batchspawner) to provide a [JupyterHub](https://jupyterhub.readthedocs.io/) [Slurm](https://slurm.schedmd.com/) Spawner with a spawn page allowing to select Slurm resources to use.
+**jupyterhub_moss** is a Python package that aims to provide a configurable [JupyterHub](https://jupyterhub.readthedocs.io/) spawner for [Slurm](https://slurm.schedmd.com/).
 
-## Example
+**jupyterhub_moss** is a Python package that aims to provide:
+- A [JupyterHub](https://jupyterhub.readthedocs.io/) [Slurm](https://slurm.schedmd.com/) Spawner that can be configured by setting the available partitions. It is an extension of [`batchspawner.SlurmSpawner`](https://github.com/jupyterhub/batchspawner).
+- An associated spawn page that changes according to the set Spawner partitions and allows the user to select Slurm resources to use.
```

Comment:
Spawner partitions ->  Slurm partitions

Reply:
Well, the spawn page changes according to the partitions that were set in the `Spawner` which can be different from the Slurm partitions...

Reply:
yet they need to correspond to Slurm partitions ;) but that's fine